styles/theme-base.css

```
@@ -555,8 +555,7 @@ div.classsynopsisinfo_comment {
 
 pre.info {
     border: 1px solid;
-    margin: 1rem 2rem 1.3rem;
-    margin-right: 0.8rem;
+    margin: 1rem 0.8rem 2rem 1.3rem;
```

Hmm, shouldn't this be:
```suggestion
    margin: 1rem 0.8rem 1.3rm 2rem;
```

---

You are right this indeed was the one. Thanks for catching this embarrassing mistake. Force-pushed with changes.